packages/jest-message-util/src/index.ts

```diff
@@ -390,9 +397,8 @@ export const separateMessageFromStack = (
   // (maybe it's a code frame instead), just the first non-empty line.
   // If the error is a plain "Error:" instead of a SyntaxError or TypeError we
   // remove the prefix from the message because it is generally not useful.
-  const messageMatch = content.match(
-    /^(?:Error: )?([\s\S]*?(?=\n\s*at\s.*:\d*:\d*)|\s*.*)([\s\S]*)$/,
-  );
+  const ERROR_REGEXP = /^(?:Error: )?([\s\S]*?(?=\n\s*at\s.*:\d*:\d*)|\s*.*)([\s\S]*)$/;
+  const messageMatch = content.match(ERROR_REGEXP);
```

Comment:
roll back changes here to reduce the diff?

Reply:
👍🏻